<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddSub</class>
 <widget class="QDialog" name="DialogAddSub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogAddSub</string>
  </property>
  <widget class="QPushButton" name="acceptPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="namelineEdit">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">key expr:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="keyExprlineEdit">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
